### PR TITLE
Apply tslint

### PR DESCRIPTION
**Pull request by 9renpoto**
# TL;DR

Connect to #447

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+    "@uncovertruth/tslint-config"
+  ],
   "rules": {
-    "no-unused-variable": true
+    "member-access": false,
+    "no-submodule-imports": false,
+    "no-console": [true, "log"],
+    "no-string-literal": false,
+    "no-shadowed-variable": false,
+    "max-classes-per-file": [true, 5],
```
Comment (shoota) [inline tslint.json]: allow 5 classes in a file.

File: tslint.json [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
-  "extends": "@uncovertruth/tslint-config",
+  "extends": [
+    "tslint-react",
+    "@uncovertruth/tslint-config"
+  ],
   "rules": {
-    "no-unused-variable": true
+    "member-access": false,
+    "no-submodule-imports": false,
+    "no-console": [true, "log"],
```
Comment (shoota) [inline tslint.json]: disallow `console.log` function. `warn` and `error` function can use.

File: packages/agent/src/global.ts [hunk 3]
```diff
@@ -54,24 +54,25 @@ export default function (
   name: string
 ) {
   const applyQueue = (delay: number) => {
+    const w = window as any
```
Comment (9renpoto) [inline packages/agent/src/global.ts]: please remove cache, see #430

**Comment by 9renpoto** [comment]
please rebase it

**Comment by shoota** [comment]
rebase with`renovate/uncovertruth-tslint-config-4.x` for v4.3.1 and fix

File: tslint.json [hunk 4]
```diff
@@ -1,7 +1,11 @@
 {
-  "extends": "@uncovertruth/tslint-config",
+  "extends": [
+    "tslint-react",
+    "@uncovertruth/tslint-config"
+  ],
   "rules": {
-    "no-unused-variable": true
+    "no-console": [true, "log"],
+    "no-var-requires": false  // for webpack.conf.ts
```
Comment (shoota) [inline tslint.json]: `import` statement not work now...

Comment (9renpoto) [reply tslint.json]: use `tslint-disable-next-line`

Comment (shoota) [reply tslint.json]: OK!!

**Pull request merged**